docs/articles/documentation/using-testcafe/command-line-interface.md

```diff
@@ -0,0 +1,225 @@
+---
+layout: docs
+title: Command Line Interface
+permalink: /documentation/using-testcafe/command-line-interface/
+---
+# Command Line Interface
+
+```sh
+testcafe [options] <browser-list-comma-separated> <file-or-glob ...>
+```
+
+## Browser List
+
+The `browser-list-comma-separated` argument specifies the list of browsers (separated by commas) where tests will be ran.
+
+### Local Browsers
+
+You can specify local browsers by using paths or [aliases](/testcafe/documentation/using-testcafe/common-concepts/browser-aliases). 
+A list of all the available aliases can be obtained by the [--list-browsers](#b) command.
+
+The following example demonstrates how to run a test in several browsers. 
+The browsers are specified differently: one by using the alias, the other by using the path.
+
+```sh
+testcafe chrome,/applications/safari.app /tests/sample-fixture.js
```

Comment:
Don't use leading slash in paths to the files. Leading slash means that your path starts from the root of FS (like with URLs). The common practice to use paths within current working directory, so it should be `tests/sample-fixture.js`. Note that for browser `/applications/safari.app` path is perfectly valid.